src/layers/core/geojson-layer/geojson-layer.js

```diff
@@ -21,52 +21,49 @@
 import {CompositeLayer, get} from '../../../lib';
 import ScatterplotLayer from '../scatterplot-layer/scatterplot-layer';
 import PathLayer from '../path-layer/path-layer';
-import PolygonLayer from '../polygon-layer/polygon-layer';
+// Use primitive layer to avoid "Composite Composite" layers for now
+import PrimitivePolygonLayer from '../primitive-polygon-layer/polygon-layer';
```

Comment:
> SimplePolygonLayer sounds better to me... no matter what name we choose, I think we should update the file names with the folder to be consistent with the new name.

I agree. Since, we already talk about simple and complex polygons (without/with holes), I called this `SolidPolygonLayer` for now and updated all names / file names accordingly.